### PR TITLE
Newsletter-291-Translate-into-French

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

File: _posts/fr/newsletters/2024-02-28-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    <td markdown="span">Nécessite un soft fork de `OP_CAT`</td>
+  </tr>
+
+  <tr>
+    <th markdown="span">Attaque de remplacement d'adresse de dernière minute</th>
+    <td markdown="span">Vulnérable</td>
+    <td markdown="span">**Non vulnérable**</td>
+    <td markdown="span">**Non vulnérable**</td>
+  </tr>
+  <th markdown="span">Retraits partiels de montant</th> <td markdown="span">Seulement si préarrangé</td>
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2024-02-28-newsletter.md:152]: @Copinmalin please revisit the table formatting here, tr and th tags

File: _posts/fr/newsletters/2024-02-28-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
+---
+titre: 'Bulletin Hebdomadaire Bitcoin Optech #291'
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2024-02-28-newsletter.md:2]: Breaks the title

```suggestion
title: 'Bulletin Hebdomadaire Bitcoin Optech #291'
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

File: _posts/fr/newsletters/2024-02-28-newsletter.md (hunk 1 above)
Comment (Copinmalin) [reply _posts/fr/newsletters/2024-02-28-newsletter.md]: OK, I hope is good this time

File: _posts/fr/newsletters/2024-02-28-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+    <td markdown="span">Nécessite un soft fork de `OP_VAULT` et [OP_CTV][topic op_checktemplateverify]</td>
+    <td markdown="span">Nécessite un soft fork de `OP_CAT`</td>
+  </tr>
+
+  <tr>
+    <th markdown="span">Attaque de remplacement d'adresse de dernière minute</th>
+    <td markdown="span">Vulnérable</td>
+    <td markdown="span">**Non vulnérable**</td>
+    <td markdown="span">**Non vulnérable**</td>
+  </tr>
```
Comment (bitschmidty) [inline _posts/fr/newsletters/2024-02-28-newsletter.md:151]:
```suggestion
  </tr>
  <tr>
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Review by bitschmidty — APPROVED** [review]
ACK, thank you @Copinmalin !

**Pull request merged**